index.js

```diff
@@ -10,7 +10,9 @@ const wc = require('with-callback')
  * @param  {arguments}   args Arguments
  * @return {undefined|*|Promise}  Promise if promisified
  */
-function promisifyCall (ctx, fn, ...args) {
+function promisifyCall (ctx, fn) {
+  const args = [];
+  args.push.apply(args, arguments) && args.splice(0,2);
```

Comment:
Hello. Thanks for the PR! This change would be useful. If you could just make the following small changes:
* This module uses [standard](http://standardjs.com/) so can you please remove the semicolons `;`'s.
* Also just put the `slice` in the next line and not as a conditional `&&`.
* Can you please add `"4.0"`, `"4.3"`, and `"4"` to `travis.yml` `node_js` property that way we'll get the proper tests run for this PR.
* Finally can you please change the `readme.hbs` and `engines` in `package.json` to specify node 4 support.